scripts/package-deb.sh

```diff
@@ -74,8 +74,11 @@ cargo deb --target "$TARGET" --deb-version "$PACKAGE_VERSION" --no-build
 
 # Rename the resulting .deb file to use - instead of _ since this
 # is consistent with our package naming scheme.
-# shellcheck disable=SC2016
-rename -v 's/vector_([^_]*)_(.*)\.deb/vector-$2\.deb/' "target/$TARGET/debian"/*.deb
+for file in target/"${TARGET}"/debian/*.deb; do
+  base=$(basename "${file}")
+  renamed=${base//_/-}
```

Comment:
This looks a bit off - the original had `([^_]*)_(.*)` into `$2`, cutting the first part (version) in addition to switching from `_` to `-`.

Reply:
Yeah, you're right, thank you. :)